docs/source/private_data_tutorial.rst

```diff
@@ -532,179 +717,193 @@ using CLI it must be base64 encoded. We use an environment variable
+to query the `assetCollection` collection as Org1:
 
-   	name = args[0]
-   	valAsbytes, err := stub.GetPrivateData("collectionMarblePrivateDetails", name) //get the marble private details from chaincode state
+.. code:: bash
 
-   	if err != nil {
-   		jsonResp = "{\"Error\":\"Failed to get private details for " + name + ": " + err.Error() + "\"}"
-   		return shim.Error(jsonResp)
-   	} else if valAsbytes == nil {
-   		jsonResp = "{\"Error\":\"Marble private details does not exist: " + name + "\"}"
-   		return shim.Error(jsonResp)
-   	}
-   	return shim.Success(valAsbytes)
-   }
+    peer chaincode query -C mychannel -n private -c '{"function":"ReadAsset","Args":["asset1"]}'
 
-Now :guilabel:`Try it yourself`
+When successful, the command will return the following result:
 
-Query for the ``name, color, size and owner`` private data of ``marble1`` as a member of Org1.
-Note that since queries do not get recorded on the ledger, there is no need to pass
-the marble name as a transient input.
+.. code:: bash
+
+    {"objectType":"asset","assetID":"asset1","color":"green","size":20,"owner":"eDUwOTo6Q049b3JnMWFkbWluLE9VPWFkbWluLE89SHlwZXJsZWRnZXIsU1Q9Tm9ydGggQ2Fyb2xpbmEsQz1VUzo6Q049Y2Eub3JnMS5leGFtcGxlLmNvbSxPPW9yZzEuZXhhbXBsZS5jb20sTD1EdXJoYW0sU1Q9Tm9ydGggQ2Fyb2xpbmEsQz1VUw=="}
+
+The `"owner"` of the asset is the identity that created the asset by invoking the smart contract. The `GetClientIdentity().GetID()` API reads the common name and issuer of the identity certificate.
+You can see that information by decoding the owner string out of base64 format:
 
 .. code:: bash
 
-    peer chaincode query -C mychannel -n marblesp -c '{"Args":["ReadMarble","marble1"]}'
+    echo eDUwOTo6Q049b3JnMWFkbWluLE9VPWFkbWluLE89SHlwZXJsZWRnZXIsU1Q9Tm9ydGggQ2Fyb2xpbmEsQz1VUzo6Q049Y2Eub3JnMS5leGFtcGxlLmNvbSxPPW9yZzEuZXhhbXBsZS5jb20sTD1EdXJoYW0sU1Q9Tm9ydGggQ2Fyb2xpbmEsQz1VUw== | base64 --decode
 
-You should see the following result:
+The result will show the common name and issuer of the owner certificate:
 
 .. code:: bash
 
-    {"color":"blue","docType":"marble","name":"marble1","owner":"tom","size":35}
+    x509::CN=org1admin,OU=admin,O=Hyperledger,ST=North Carolina,C=US::CN=ca.org1.example.com,O=org1.example.com,L=Durham,ST=North Carolina,C=US    
+
 
-Query for the ``price`` private data of ``marble1`` as a member of Org1.
+Query for the ``appraisedValue`` private data of ``asset1`` as a member of Org1.
 
 .. code:: bash
 
-    peer chaincode query -C mychannel -n marblesp -c '{"Args":["ReadMarblePrivateDetails","marble1"]}'
+    peer chaincode query -C mychannel -n private -c '{"function":"ReadAssetPrivateDetails","Args":["Org1MSPPrivateCollection","asset1"]}'
 
 You should see the following result:
 
 .. code:: bash
 
-    {"docType":"marblePrivateDetails","name":"marble1","price":99}
+    {"assetID":"asset1","appraisedValue":100}
 
 .. _pd-query-unauthorized:
 
 Query the private data as an unauthorized peer
```

Comment:
This probably does not need to be a top level section any more. I would focus on the transfer and have this as a sub-section

Reply:
Done, moved out of top level.